classes/class-gf-securesubmit.php

```diff
@@ -714,7 +716,7 @@ public function maybe_validate($validationResult) {
                 continue;
             }
 
-            if (empty($this->validateACH()) && $this->getSecureSubmitJsError() && $this->hasPayment($validationResult)) {
+            if (false == $this->validateACH() && $this->getSecureSubmitJsError() && $this->hasPayment($validationResult)) {
```

Comment:
That being said, I can replace this with a two part solution, create a temporary variable thun use `empty` as before